bpf/lib/policy.h

```diff
@@ -30,62 +30,65 @@ policy_sk_egress(__u32 identity, __u32 ip,  __u16 dport)
 	if (!map)
 		return CTX_ACT_OK;
 
+	/* Policy match precedence:
```

Comment:
Maybe worth ditching the first commit all together, and keeping this one with the updated comments?